Modules/_datetimemodule.c

```diff
@@ -3397,7 +3403,7 @@ date_fromisocalendar(PyObject *cls, PyObject *args, PyObject *kw)
     }
 
     if (rv == -3) {
-        PyErr_Format(PyExc_ValueError, "Invalid day: %d (range is [1, 7])",
+        PyErr_Format(PyExc_ValueError, "Invalid weekday: %d (range is [1, 7])",
```

Comment:
Should this get normalized too? It's the only message here that doesn't follow the `x must by y, but got z`

Reply:
This error is related to **weekday** and is not related to **day** errors with other message.